src/OnlineRetailPortal.Web/Models/AddProductRequest.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace OnlineRetailPortal.Web
+{
+    public class AddProductRequest
+    {
+        public string UserId { get; set; }
```

Comment:
How will you differentiate between user id passed in header and user id taken in request.
Shouldn’t the field name be postedBy or SellerId?
Just a question

Reply:
done